sunbird/sample1.py

```diff
@@ -0,0 +1,5395 @@
+    for row_index_env in range(2, tasksDetailsSheet.nrows):
+        dictTasksDetails = {keysTasks[col_index_env]: tasksDetailsSheet.cell(row_index_env, col_index_env).value
+                            for col_index_env in range(tasksDetailsSheet.ncols)}
+        taskName = str(dictTasksDetails["TaskTitle"]).encode('utf-8').decode('utf-8').strip()
+        # subtaskname = str(dictTasksDetails["Subtask"]).encode('utf-8').decode('utf-8').strip()
+
+        if dictTasksDetails['TaskId'] :
+           taskId = str(dictTasksDetails["TaskId"]).encode('utf-8').decode('utf-8').strip() + "-" + str(millisecond)
+           taskminNoOfSubmissionsRequired = str(dictTasksDetails["Number of submissions for observation"]).strip()
+           sequenceNumber = sequenceNumber + 1
+           taskSolutionType = ""
+           try:
+               taskDescription = str(dictTasksDetails["description"]).strip()
+           except:
+               taskDescription = ""
+           if dictTasksDetails["observation Name"] != "":
+               taskType = "observation"
+           elif dictTasksDetails["learningResources1-name"] != "" and dictTasksDetails["learningResources1-link"] != "":
+               taskType = "content"
+           else:
+               taskType = "simple"
+
+           hasAParentTask = "NO"
+           parentTaskOperator = ""
+           parentTaskValue = ""
+           parentTaskId = ""
+
+           if dictTasksDetails["observation Name"] != "":
+                pass
+            #    solutionNameOrId = dictTasksDetails["observation Name"].encode('utf-8').decode('utf-8')
+            #    taskSolutionType = "observation"
+            #    solutionDetailsInTask = checkEntityOfSolution(projectName_for_folder_path, solutionNameOrId, accessToken)
+            #    solutionSubType = solutionDetailsInTask[0]
+            #    solutionId = solutionDetailsInTask[1]
+
+
+            #    projectUpload = pd.read_csv(projectFilePath + "projectUpload.csv")
+            #    # updating the column value/data
+            #    projectUpload.loc[0, 'entityType'] = solutionDetailsInTask[0]
+
+            #    # writing into the file
+            #    projectUpload.to_csv(projectFilePath + "projectUpload.csv", index=False)
+           else:
+               solutionId = ""
+               taskSolutionType = ""
+               solutionSubType = ""
+
+           if str(dictTasksDetails["Mandatory task(Yes or No)"]).strip().strip().lower() == "no":
+               isDeletable = "TRUE"
+           else:
+               isDeletable = "FALSE"
+           task_values = [taskName, taskId, taskDescription, taskType, hasAParentTask, parentTaskOperator, parentTaskValue,
+                          parentTaskId, taskSolutionType, solutionSubType, solutionId, isDeletable]
+           task_lr_value_count = 1
+           for task_lr in range(0, int(taskLearningResource_count)):
+               task_lr_name = str(dictTasksDetails["learningResources" + str(task_lr_value_count)
```

Comment:
now we have 2 different approaches. 
1 with program file
2 with project file.

SO i reccommend instead of having the command line argument with same name tag, use different for different approaches. 
ex : --programFile and --resourceFile
Identify the approach from the command line argument passed.